test/distributed/org/apache/cassandra/distributed/test/InternodeEncryptionEnforcementTest.java

```diff
@@ -432,7 +426,8 @@ public static class AllowFirstAndRejectOtherOutboundAuthenticator extends Reject
         @Override
         public boolean authenticate(InetAddress remoteAddress, int remotePort, Certificate[] certificates, InternodeConnectionDirection connectionType)
         {
-            if (connectionType == InternodeConnectionDirection.OUTBOUND)
+            if (connectionType == InternodeConnectionDirection.OUTBOUND ||
+                connectionType == InternodeConnectionDirection.OUTBOUND_PRECONNECT)
```

Comment:
why add `OUTBOUND_PRECONNECT`? Doesn't that mean you don't exercise the outbound auth failure path as it always preconnects before the real auth attempt.